sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveShim.scala

```diff
@@ -841,5 +841,6 @@ private[client] class Shim_v1_2 extends Shim_v1_1 {
       case e: InvocationTargetException => throw e.getCause()
     }
   }
-
+  
 }
+
```

Comment:
yes... I make a mistake, there is another HiveShim.scala .....
sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveShim.scala